### PR TITLE
{cae}[intel/2015b] SWASH 3.14 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/s/SWASH/SWASH-3.14-intel-2015b-mpi.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+easyblock = 'MakeCp'
+
+name = 'SWASH'
+version = '3.14'
+versionsuffix = '-mpi'
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/SWASH/SWASH-3.14-intel-2015b-mpi.eb]: don't we usually drop the suffix for MPI and add `-mt` for non-mpi?

Comment (boegel) [inline easybuild/easyconfigs/s/SWASH/SWASH-3.14-intel-2015b-mpi.eb]: I have all three variants here (including serial), so I want to make it very clear which is which.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/49fdb820b319df639ebb for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5d622027296907d7e68d for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**